bundles/AdminBundle/public/js/pimcore/element/helpers/gridColumnConfig.js

```diff
@@ -943,7 +943,7 @@ pimcore.element.helpers.gridColumnConfig = {
         }
 
         var fields = this.getGridConfig().columns;
-        var fieldKeys = Object.keys(fields);
+        var fieldKeys = Object.entries(fields).map(([key, value]) => value.fieldConfig?.label || key);
```

Comment:
Hi @SamyMP, can I ask you why is this change necessary? :)

Reply:
This is fixed, sorry about that

Reply:
No worries, thank you for the fix :)